src/pages/projectSettings/index.tsx

```diff
@@ -36,10 +37,11 @@ const { validateObjectId } = validators;
 const ProjectSettings: React.FC = () => {
   usePageTitle(`Project Settings`);
   const dispatchToast = useToastContext();
-  const { projectIdentifier: identifier, tab } = useParams<{
-    projectIdentifier: string;
-    tab: ProjectSettingsTabRoutes;
-  }>();
+  const { [slugs.projectIdentifier]: identifier, [slugs.tab]: tab } =
+    useParams<{
+      [slugs.projectIdentifier]: string | null;
```

Comment:
just curious if `| null` is required here:
```suggestion
      [slugs.projectIdentifier]: string;
```